subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/ivyresolve/verification/ChecksumAndSignatureVerificationOverride.java

```diff
@@ -189,6 +199,17 @@ public void artifactsAccessed(String displayName) {
                     // the else is just to avoid telling people to use `--write-verification-metadata` if we suspect compromised dependencies
                     formatter.node("If the dependency is legit, update the gradle/dependency-verification.xml manually (safest) or run with the --write-verification-metadata flag (unsecure).");
                 }
+                if (!affectedFiles.isEmpty()) {
+                    formatter.blankLine();
+                    formatter.node("For your information here are the path to the files which failed verification:");
```

Comment:
`are the path to the files`

Does that sound right?

Reply:
Grammar error, will tweak it.